src/runtime-rs/crates/resource/src/rootfs/nydus_rootfs.rs

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SP&DX-License-Identifier: Apache-2.0
+//
+
+use super::{Rootfs, TYPE_OVERLAY_FS};
+use crate::{
+    rootfs::{HYBRID_ROOTFS_WRITABLE_LAYER_LOWER_DIR, ROOTFS},
+    share_fs::{
+        do_get_guest_path, do_get_guest_virtiofs_path, get_host_rw_shared_path, rafs_mount,
+        PASSTHROUGH_FS_DIR,
+    },
+};
+use agent::Storage;
+use anyhow::{anyhow, Context, Result};
+use async_trait::async_trait;
+use hypervisor::Hypervisor;
+use kata_sys_util::mount;
+use kata_types::mount::{Mount, NydusExtraOption};
+use std::fs;
```

Comment:
I think it's better to move `use std::fs` to line 6 before other packages and add a space.

Reply:
fixed